cypress/e2e/release-gate/landing-page.cy.ts

```diff
@@ -16,9 +266,10 @@ describe('Landing page', () => {
   });
 
   it('tooltip is shown when hovering over the gear/question icon', () => {
+    cy.visit('/');
     cy.login();
-
     cy.visit('/');
+    cy.reload();
```

Comment:
Why is a reload necessary here? A one-line comment might help explain.

Reply:
For reference for anyone who reads this, my internship ended so I can't change any of this if I wanted to... That being said, the `.reload()` is there because sometimes, when not reloaded, the DOM tree fails to render correctly, which fails some of the tests